TestsCalculator/CalculatorPageTests.cs

```diff
@@ -145,8 +408,8 @@ public void CheckMonthInDropdown()
         {
             //Arrange
             IWebElement monthDropdown = driver.FindElement(By.Id("month"));
-            //Act
-            //object of SelectElement
+
```

Comment:
For such test you can hard-code the expected values:
```suggestion
        // Arrange
        List<string> expectedMonths = new List<string> { "January", "February", "December" };
```